src/PSPackageProject.psm1

```diff
@@ -654,11 +654,12 @@ Describe "Test ${moduleName}" {
     # make pspackageproject.json
     $jsonPrj =
     @{
-        SourcePath = 'src'
+        SourcePath = Join-Path $moduleRoot "src"
```

Comment:
We do not want to set full paths here. They will not make any sense in CI. My PR #13 sets them dynamically when `Get-PSPackageProjectConfig` is called.